libnavigation-ui/src/main/java/com/mapbox/navigation/ui/speed/SpeedLimitView.java

```diff
@@ -0,0 +1,213 @@
+package com.mapbox.navigation.ui.speed;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.graphics.Color;
+import android.graphics.drawable.GradientDrawable;
+import android.graphics.drawable.LayerDrawable;
+import android.text.SpannableStringBuilder;
+import android.text.style.RelativeSizeSpan;
+import android.text.style.StyleSpan;
+import android.util.AttributeSet;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.constraintlayout.widget.ConstraintLayout;
+import androidx.lifecycle.LifecycleObserver;
+
+import com.mapbox.navigation.base.speed.model.SpeedLimit;
+import com.mapbox.navigation.base.speed.model.SpeedLimitSign;
+import com.mapbox.navigation.base.speed.model.SpeedLimitUnit;
+import com.mapbox.navigation.ui.R;
+
+import static android.graphics.Typeface.BOLD;
+import static android.text.Spanned.SPAN_EXCLUSIVE_EXCLUSIVE;
+
+public class SpeedLimitView extends ConstraintLayout implements LifecycleObserver {
+
+  private static final double KILO_MILES_FACTOR = 0.621371;
+  private static final float RADIUS = 10f;
+  private static final int VIENNA_MUTCD_BORDER_INSET = 4;
+  private static final int MUTCD_INNER_BACKGROUND_INSET = 7;
+  private static final int VIENNA_INNER_BACKGROUND_INSET = 16;
+  private static final int VIENNA_MUTCD_OUTER_BACKGROUND_INSET = 0;
```

Comment:
Based on the screenshots in https://github.com/mapbox/mapbox-navigation-android/pull/3838#issue-529783139, these metrics result in an appearance that doesn’t look quite as tight as the iOS implementation seen in the screenshots in mapbox/mapbox-navigation-ios#2291. Maybe that comes down to a difference in font metrics or something else, but the main issue is that the MUTCD sign looks more squat than it should, and the Vienna sign doesn’t look round enough.

Reply:
I agree with @1ec5 - The signs don't quite look right. We should match the designs on the two platforms and I would argue that the MUTCD sign on iOS is a good target for Android. If people want to debate that the MUTCD sign on Android is better than iOS then I think we can discuss that, but the Vienna sign is definitely wrong and should be a simple circle, not an oval.